var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -35,7 +35,7 @@ class SuiteSparse(Package):
     version('4.5.3', '8ec57324585df3c6483ad7f556afccbd')
     version('4.5.1', 'f0ea9aad8d2d1ffec66a5b6bfeff5319')
 
-    variant('tbb', default=True, description='Build with Intel TBB')
```

Comment:
I don't think `tbb` is Intel specific, i.e. you can compile and use it with `clang` without issues. If there something special about `xl` compiler, please add a note/comment above why `tbb` needs to be disabled.